code/index.html

```diff
@@ -3,45 +3,68 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>Guess Who?</title>
     <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Black+Ops+One&display=swap"
       rel="stylesheet"
     />
+    <link rel="shortcut icon" href="./lego.png" type="image/x-icon" />
   </head>
+
   <body>
     <aside class="question-section">
       <button id="restart" class="outlined-button">RESTART</button>
-
-      <img
-        class="guess-who-icon"
-        src="images/guess-who-bubble.png"
-        alt="Guess Who"
-      />
-
+      <img class="guess-who-icon" src="./Legologogreen.png" alt="Guess Who" />
       <h1>Does the person have</h1>
 
       <select id="questions">
         <optgroup label="hair">
+          <option value="bald">no hair</option>
+          <option value="black">black hair</option>
+          <option value="blond">blond hair</option>
           <option value="brown">brown hair</option>
-          <option value="yellow">yellow hair</option>
-          <!-- Add the rest of hair colors as options  -->
+          <option value="gray">grey hair</option>
+          <option value="hidden">hidden hair</option>
+          <option value="white">white hair</option>
         </optgroup>
-        <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
-          <!-- Add the rest of eye colors as options  -->
+
+        <optgroup label="headpiece">
+          <option value="hat">hat</option>
+          <option value="helmet">helmet</option>
+          <option value="no headpiece">no headpiece</option>
         </optgroup>
+
         <optgroup label="accessories">
+          <option value="cape">cape</option>
           <option value="glasses">glasses</option>
-          <!-- Add the other accessory option here. (hat)  -->
+          <option value="pet">pet</option>
+          <option value="weapon">weapon</option>
         </optgroup>
+
         <optgroup label="other">
-          <option value="smoker">a smoking habit</option>
+          <option value="superhero">superhero</option>
```

Comment:
I really love this, both that you adding new options to the game and also the villain/superhero part!